api_notifications.yaml

```diff
@@ -54,25 +54,25 @@ paths:
           description: There was an error in forwarding the notification to the Notification Hub.
 definitions:
   CreatedMessageWithContent:
-    $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/v0.47.0/api/definitions.yaml#/CreatedMessageWithContent"
+    $ref: "https://raw.githubusercontent.com/teamdigitale/digital-citizenship-functions/master/api/definitions.yaml#/CreatedMessageWithContent"
```

Comment:
we should always refer to a specific version (ideally the one that's deployed), never to `master`, `master` makes builds not replicable